vip-helpers/vip-roles.php

```diff
@@ -88,6 +88,11 @@ function wpcom_vip_override_role_caps( $role, $caps ) {
 
 	$role_obj->capabilities = (array) $caps;
 
+	$roles = wp_roles();
+
+	$roles->roles[ $role ]['capabilities'] = $caps;
+	update_option( $roles->role_key, $roles->roles );
```

Comment:
There was an option to use `remove_cap()` and `add_cap()` in a loop instead; however, I did not like that because it would trigger `update_option()` for every operation, and that does not look database- and cache-friendly.
_